ca/certificates.go

```diff
@@ -444,7 +444,10 @@ func NewRootCA(rootCertBytes, signCertBytes, signKeyBytes []byte, certExpiry tim
 		pool.AddCert(cert)
 	}
 
-	// Calculate the digest for our Root CA bundle
+	// Make sure the root certs are normalized before getting the digest, so that we can generate a consistent digest
+	// even if the root certs are.  Also store the normalized root cert in the RootCA object.
+	rootCertBytes = NormalizePEMs(rootCertBytes)
```

Comment:
Yes, you're right.  I think maybe it's ok that we calculate new CA cert hashes Do you think it's ok that we calculate the new hash with the normalized PEM, or should we leave everything as was before and just do the normalization when we compare bytes?